package.json

```diff
@@ -79,8 +79,7 @@
     "pre-commit": "lint-staged"
   },
   "lint-staged": {
-    "*.js": "trunk check --fix --filter=eslint",
-    "*.md": "trunk check --fix --filter=markdownlint",
+    "*": "trunk check --fix",
     "lib/rules/*.js": [
       "node tools/update-eslint-all.js",
```

Comment:
Sorry for coming here so late. Have we checked this https://github.com/eslint/eslint/pull/19354#discussion_r1924000139? 

`node tools/update-eslint-all.js`  creates a new file which will be run by `trunk check --fix`. Which runs first? is it the script or format?